rbkit-lib/ui/actiontoolbar.h

```diff
@@ -35,8 +39,9 @@ class ActionToolbar : public QObject
     Subscriber *subscriber;
     ConnectionStates connectionState;
     QThread subscriberThread;
+    AppMainwindow *window;
```

Comment:
we are not instantiating AppMainwindow here
this variable will just hold a reference to mainwindow so that some of the function which need to call methods on mainwindow can call them